.circleci/config.yml

```diff
@@ -513,12 +518,19 @@ jobs:
           name: Add python to bash path
           command: echo "export PATH=\"$PATH:/c/python27amd64/\"" >> $BASH_ENV
       - build-upstream
-  build-upstream-mac:
-    macos:
-      xcode: "9.0"
+  test-upstream-other-windows:
     environment:
+      PYTHON_BIN: "/c/python27amd64/python.exe"
+      PYTHONUNBUFFERED: "1"
```

Comment:
what does this do?

Reply:
Hmm I can't remember but it was needed on the original windows builder, I'm just duplicating the environment from the builder into the tester.